cfme/tests/containers/test_containers_summary.py

```diff
@@ -22,7 +22,9 @@
     pytest.mark.tier(1)]
 pytest_generate_tests = testgen.generate([ContainersProvider], scope='function')
 
-tested_objects = [Service, Route, Project, Pod, Image, Container, ImageRegistry, Node]
+# TODO Update test to use new Node views and widgets
+tested_objects = [Service, Route, Project, Pod, Image, Container, ImageRegistry  # , Node
+                  ]
```

Comment:
**REQUIRED**
It looks ugly, please add `Node` into TODO section

Reply:
:+1: